src/net/udns/udns_resolver.c

```diff
@@ -1241,7 +1241,7 @@ struct dns_resolve_data {
   void *dnsrd_result;
 };
 
-static void dns_resolve_cb([[maybe_unused]] struct dns_ctx *ctx, void *result, void *data) {
+static void dns_resolve_cb(__attribute__((unused)) struct dns_ctx *ctx, void *result, void *data) {
```

Comment:
Not going back to using `__attribute__`.

Reply:
This is actually an issue that C compilers don’t support attributes until C23.